manifests/4.5/sriov-network-operator-sriovnetworknodestate.crd.yaml

```diff
@@ -108,25 +106,28 @@ spec:
                     type: string
                   linkSpeed:
                     type: string
+                  mac:
```

Comment:
why there is a `mac` field? do we add this in nodestate somewhere?

Reply:
It is here https://github.com/openshift/sriov-network-operator/blob/160307f84f7f91d9cd193d063f11350d909617a8/pkg/apis/sriovnetwork/v1/sriovnetworknodestate_types.go#L35
But, currently, we didn't use this parameter at all.

Reply:
shall we remove it if not used?

Reply:
I think we can keep it for now. I plan to implement this function later.